### DISCUSSION TITLE
docs/certmanager/

### DISCUSSION THREAD

**Discussion by giscus** [ricsanfre/pi-cluster#259] [Announcements]
# docs/certmanager/

How to deploy a centralized TLS certificates management solution based on Cert-manager in our Raspberry Pi Kuberentes cluster.

https://picluster.ricsanfre.com/docs/certmanager/

**Comment by bpartin2009** [comment]
When using self-signed certs and setting enable_letsencrypt to false, the external_services.yml line 65 pulls in generate_custom_ca.yml
The generate_custom_ca.yml uses openssl modules that are not found in the ansible-runner docker image running on the Ansible Control Node.

**Reply by ricsanfre** [reply]
Hi @bpartin2009. Could you please open a new [issue in github](https://github.com/ricsanfre/pi-cluster/issues), so we can track it and fix it?
Thanks

**Comment by benbourner** [comment]
Noticed two issues with the self-signed flow described here:
1) Cluster Issuer is named "self-signed-issuer" but custom root certificate refers to "my-self-signed-issuer"
2) Previously described Traefik install refers to "ca-issuer" but its defined here as "my-ca-issuer"

**Reply by ricsanfre** [reply]
Thanks @benbourner, I have updated the document fixing those issues.
